src/lib.rs

```diff
@@ -283,8 +291,8 @@ impl Watch {
     ///
     /// Workspace's `target` directory and hidden paths are excluded by default.
     pub fn run(mut self, commands: impl Into<CommandList>) -> Result<()> {
-        let commands = commands.into();
         let metadata = metadata();
+        let commands = self.include_commands(commands);
```

Comment:
This code isnt really complex. You can put it all in there instead of making a separate function, so....

Reply:
...so the code that comes after doesn't feel lonely

Reply:
I have a deja vu about this conversation